resources/views/includes/script.blade.php

```diff
@@ -7,6 +7,19 @@ function bootstrapSupportBubble(element) {
         const responseContainer = element.querySelector('.spatie-support-bubble__response');
         const errorMessage = element.querySelector('.spatie-support-bubble__error');
 
+        let closedClassList;
```

Comment:
The only difference between the two cases now is that `-` that gets prefixed.

Things could be simplified by storing `translate-x-full` in a variable, prefix it when needed, and than using the variable.

Could you refactor this?

Reply:
Will Do Sir.